### PR TITLE
Change sectioned form error indicator to be a status indicator with proper left padding

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-sectioned-form-error-indicator-fix](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-sectioned-form-error-indicator-fix)

**Review by Blackbaud-ToddRoberts — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-sectioned-form-error-indicator-fix](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-sectioned-form-error-indicator-fix)

File: src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts [hunk 1]
```diff
@@ -291,9 +291,8 @@ describe('Sectioned form component', () => {
 
     // check section is required
     tabs = el.querySelectorAll('sky-vertical-tab');
```
Comment (johnhwhite) [inline src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts:293]: This variable is no longer needed, right?

File: src/app/public/modules/vertical-tabset/vertical-tab.component.scss [hunk 2]
```diff
@@ -91,6 +91,11 @@
       border-left: 3px solid $sky-highlight-color-info;
       padding-left: 7px;
     }
+
+
```
Comment (johnhwhite) [inline src/app/public/modules/vertical-tabset/vertical-tab.component.scss:95]: There's an extra blank line here.

File: src/app/visual/sectioned-form/sectioned-form-visual.component.ts [hunk 3]
```diff
@@ -7,4 +8,10 @@ import { ChangeDetectionStrategy, Component} from '@angular/core';
 })
 export class SectionedFormVisualComponent {
   public maintainSectionContent: boolean = false;
+
+  constructor(@Optional() private themeSvc: SkyThemeService) {}
+
+  public themeSettingsChange(themeSettings: SkyThemeSettings): void {
+    this.themeSvc.setTheme(themeSettings);
```
Comment (johnhwhite) [inline src/app/visual/sectioned-form/sectioned-form-visual.component.ts:15]: Since theme service is optional, this should check if `this.themeSvc` is set.

**Review by johnhwhite — CHANGES_REQUESTED** [review]
Couple small things.

File: src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/sectioned-form/sectioned-form.component.spec.ts:293]: The first use of it is still needed -but you are right that it doesn't need set here again. I've removed that.

File: src/app/public/modules/vertical-tabset/vertical-tab.component.scss (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/vertical-tabset/vertical-tab.component.scss:95]: Good catch - fixed.

File: src/app/visual/sectioned-form/sectioned-form-visual.component.ts (hunk 3 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/visual/sectioned-form/sectioned-form-visual.component.ts:15]: Yup - good call - fixed.

**Review by johnhwhite — APPROVED** [review]
🚢

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-sectioned-form-error-indicator-fix](https://developer.blackbaud.com/skyux-tabs?_sp.spa=pr-sectioned-form-error-indicator-fix)

**Pull request merged**